ui/app/store/actions.js

```diff
@@ -53,7 +52,6 @@ const actions = {
   TRANSITION_FORWARD: 'TRANSITION_FORWARD',
   TRANSITION_BACKWARD: 'TRANSITION_BACKWARD',
   transitionForward,
-  transitionBackward,
```

Comment:
`TRANSITION_BACKWARD` can be removed as well, along with associated code.

Reply:
The same goes for these as well:
* `SHOW_CREATE_VAULT`
* `SHOW_RESTORE_VAULT`
* `SHOW_INIT_MENU`
* `SHOW_NEW_ACCOUNT_PAGE`
* `CREATE_NEW_VAULT_IN_PROGRESS`
* `NEW_ACCOUNT_SCREEN`
* `REQUEST_ACCOUNT_EXPORT`
* `EXPORT_ACCOUNT`
* `NEXT_TX`
* `EDIT_TX`
* `PREVIOUS_TX`

... probably many more, that's as far as I got.

Reply:
Maybe you intended to do this in a separate, follow-up PR? Let me know if that was your intention

Reply:
That was not the intention, I'll remove those as well.

Reply:
Updated

Reply:
A bunch of these have associated Redux code that's now dead as well